### DISCUSSION TITLE
Allow for clicking outside selection view to dismiss it

### DISCUSSION THREAD

**Discussion by r-c-k** [iotaledger/firefly#1784] [Features / Enhancements]
<img width="888" alt="Bildschirmfoto 2021-12-18 um 07 01 43" src="https://user-images.githubusercontent.com/17240275/146631164-ce352033-fc24-4290-a639-9539926e7592.png">

Allow the user to click outside of a list view to dismiss it / abort the selection.

**Comment by rafaeldjpbrochado** [comment]
I believe this is a duplicate of: https://github.com/iotaledger/firefly/discussions/1553

Still a good suggestion though.

**Reply by nicole-obrien** [reply]
Slightly different, so will keep this open.
